pages/[...slug].tsx

```diff
@@ -44,6 +44,7 @@ export interface PaginationType {
 interface Props {
     frontMatter: {
         title: string;
+        subtitle: string;
```

Comment:
```suggestion
        subtitle?: string;
```
With this change it becomes optional. Rest looks good

Reply:
Making this change